cmake/RunCoverage.cmake.in

```diff
@@ -31,16 +31,20 @@
 # ```
 
 if($<BOOL:$<CONFIG:Debug>>)
-    get_filename_component(root_dir "${CMAKE_CURRENT_LIST_DIR}" DIRECTORY)
+    set(source_dir "${CMAKE_ARGV0}")
```

Comment:
Might want to add a comment at the top of the script that it expects the source_dir and binary_dir as arguments in that order.